### PR TITLE
Use general neofs-aio container

### LINKED ISSUE #92: Use current neofs-aio versions for tests
We're testing with obsolete neofs-aio-testcontainer image, while after nspcc-dev/neofs-aio#43 we should be able to use standard neofs-aio. So let's try it to have the gateway tested using current NeoFS versions.

### PR DESCRIPTION
**Pull request by smallhive**
close #92

### CODE REVIEW

**Comment by smallhive** [comment]
I consider some code with `tickEpoch` maybe removed after [this issue](https://github.com/nspcc-dev/neofs-aio/issues/54) fixed

File: cmd/neofs-rest-gw/integration_test.go [hunk 1]
```diff
@@ -79,16 +89,16 @@ func runLocalTests(ctx context.Context, t *testing.T, key *keys.PrivateKey) {
 }
 
 func runTestInContainer(rootCtx context.Context, t *testing.T, key *keys.PrivateKey) {
-	aioImage := "nspccdev/neofs-aio-testcontainer:"
```
Comment (roman-khimov) [inline cmd/neofs-rest-gw/integration_test.go:82]: Let's keep 0.34.0 version of `nspccdev/neofs-aio-testcontainer` and add `neofs-aio` with 0.36.0/0.36.1.

File: cmd/neofs-rest-gw/integration_test.go [hunk 2]
```diff
@@ -62,6 +63,15 @@ const (
 	useLocalEnvironment = false
 )
 
+var (
+	tickEpoch = []string{
```
Comment (roman-khimov) [inline cmd/neofs-rest-gw/integration_test.go:67]: OK to have for now, eventually we'll drop it.

File: cmd/neofs-rest-gw/integration_test.go [hunk 3]
```diff
@@ -131,20 +141,38 @@ func runTests(ctx context.Context, t *testing.T, key *keys.PrivateKey, node stri
...
-		Image:       image,
-		WaitingFor:  wait.NewLogStrategy("aio container started").WithStartupTimeout(30 * time.Second),
-		Name:        "aio",
-		Hostname:    "aio",
-		NetworkMode: "host",
+		Image:      image,
+		WaitingFor: wait.NewLogStrategy("aio container started").WithStartupTimeout(30 * time.Second),
+		Name:       "restgw-aio-test-" + version,
+		Hostname:   "aio",
+		Env: map[string]string{
```
Comment (roman-khimov) [inline cmd/neofs-rest-gw/integration_test.go:150]: Please create an issue for AIO, it should start with these default, unless they're overridden somehow.

**Review by roman-khimov — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**